server/protocols/jmap-integration-testing/memory-jmap-integration-testing/src/test/resources/logback-test.xml

```diff
@@ -6,7 +6,7 @@
         </encoder>
     </appender>
 
-    <root level="WARN">
+    <root level="ERROR">
```

Comment:
I thought we agreed to keep warns logs in tests. If you think we should remove some, we should move them from WARN to INFO where they are written.

Reply:
True. Actually I turned logging down to better see my execution timings while working on this issue. It look like I commited some by mistake.